doc/classes/Node3D.xml

```diff
@@ -18,32 +18,33 @@
 			<return type="void" />
 			<param index="0" name="gizmo" type="Node3DGizmo" />
 			<description>
-				Attach an editor gizmo to this [Node3D].
-				[b]Note:[/b] The gizmo object would typically be an instance of [EditorNode3DGizmo], but the argument type is kept generic to avoid creating a dependency on editor classes in [Node3D].
+				Attaches the given [param gizmo] to this node. Only works in the editor.
+				[b]Note:[/b] [param gizmo] should be an [EditorNode3DGizmo]. The argument type is [Node3DGizmo] to avoid depending on editor classes in [Node3D].
 			</description>
 		</method>
 		<method name="clear_gizmos">
 			<return type="void" />
 			<description>
-				Clear all gizmos attached to this [Node3D].
+				Clears all [EditorNode3DGizmo] objects attached to this node. Only works in the editor.
 			</description>
 		</method>
 		<method name="clear_subgizmo_selection">
 			<return type="void" />
 			<description>
-				Clears subgizmo selection for this node in the editor. Useful when subgizmo IDs become invalid after a property change.
+				Deselects all subgizmos for this node. Useful to call when the selected subgizmo may no longer exist after a property change. Only works in the editor.
 			</description>
 		</method>
 		<method name="force_update_transform">
 			<return type="void" />
 			<description>
-				Forces the transform to update. Transform changes in physics are not instant for performance reasons. Transforms are accumulated and then set. Use this if you need an up-to-date transform when doing physics operations.
+				Forces the node's [member transform] to update. Fails if the node is not inside the tree.
```

Comment:
I took a peek at the implementation, I think this would be more accurate.

```suggestion
				Forces the [SceneTree]'s list of pending transform notifications to include this node, which will be accumulated and applied [i]once[/i] at the end of the frame. Fails if the node is not inside the tree.
```

I did copy some text from the line below, I'm not exactly sure what's the best phrasing for what goes in a note and what goes here, but anything is fine to me.

Reply:
Mentioning the scene tree's inner workings on this is a bit much, but the behavior detail (possibly akin to `queue_redraw()`) may be very important.
But I'm not sure if it's going to be misunderstood or not. `force_update_transform` implies the updating happens in the moment the method is called, but it doesn't seem true, then?

Reply:
Okay, hold on.

> Forces the [SceneTree]'s list of pending transform notifications to include this node, which will be accumulated and applied [i]once[/i] at the end of the frame.

https://github.com/godotengine/godot/blob/44c55ad59b01dbf360777b1a9a18a229aeb1f4f7/scene/3d/node_3d.cpp#L828-L836

What you described is the usual behaviour. The method seems to do exactly the opposite. That is, remove them from the list of "pending notifications" and send NOTIFICATION_TRANSFORM_CHANGED right away, which makes more sense for its namesake.

Reply:
Oh, you're right.